public/app/features/dimensions/types.ts

```diff
@@ -51,6 +51,21 @@ export interface ScaleDimensionOptions {
   hideRange?: boolean; // false
 }
 
+export enum ScalerDimensionMode {
+  Mod = 'mod',
+  Capped = 'capped',
+}
```

Comment:
What do these modes mean in context? (i.e. how do they impact UX)

Reply:
Exposing to the user how an angle, or field value that will be used as an angle, is treated when beyond the minumum and maximum limits. There are some examples in the test file

Reply:
"clamped" is also commonly used for this terminology. not sure if there are any other examples in the codebase to maintain uniformity, but might be worth checking.

Reply:
"limited" or "bounded" or "constrained" are some other options